core/src/main/java/io/grpc/CallOptions.java

```diff
@@ -170,6 +175,18 @@ public CallOptions withAffinity(Attributes affinity) {
   }
 
   /**
+   * Sets whether the call is not
+   * <a href="https://github.com/grpc/grpc/blob/master/doc/fail_fast.md">fail fast</a>.
+   * 'Fail fast' is the default option for gRPC calls and 'Wait for ready' is the opposite to it.
+   */
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1915")
+  public CallOptions withWaitForReady(boolean waitForReady) {
```

Comment:
It was mentioned before that C++ would use yet another term for this feature. We may not want to introduce yet another term for the same thing. We should either use the same term as C++ or just "fail-fast".